sdk/ml/azure-ai-ml/azure/ai/ml/_schema/monitoring/monitor_definition.py

```diff
@@ -37,6 +38,7 @@ class MonitorDefinitionSchema(metaclass=PatchedSchemaMeta):
                 NestedField(CustomMonitoringSignalSchema),
                 NestedField(GenerationSafetyQualitySchema),
                 NestedField(ModelPerformanceSignalSchema),
+                NestedField(GenerationTokenStatisticsSchema)
```

Comment:
Make sure the token statistics class is also included in the monitor definition entity

Reply:
added to definition